quill-core/src/main/scala/io/getquill/util/Messages.scala

```diff
@@ -4,6 +4,11 @@ import scala.reflect.macros.blackbox.{ Context => MacroContext }
 
 object Messages {
 
+  private val debugEnabled =
```

Comment:
@schmitch This would disable by default one of the most important features that Quill was designed to provide: compile time feedback of the generated query. I still think it should be *enabled* by default. @getquill/maintainers wdyt?